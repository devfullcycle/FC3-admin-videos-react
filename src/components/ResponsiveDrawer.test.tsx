import { renderWithProviders, screen } from "../utils/test-utils";
import ResponsiveDrawer from "./ResponsiveDrawer";

describe("ResponsiveDrawer", () => {
  it("should render correctly", () => {
    const { asFragment } = renderWithProviders(
      <ResponsiveDrawer onClose={() => {}} open={false} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
