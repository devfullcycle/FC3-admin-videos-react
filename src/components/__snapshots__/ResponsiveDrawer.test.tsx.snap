// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponsiveDrawer should render correctly 1`] = `
<DocumentFragment>
  <nav
    class="MuiBox-root css-1o5pq7n"
  >
    <div
      class="MuiDrawer-root MuiDrawer-docked css-aasjdk-MuiDrawer-docked"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiDrawer-paper MuiDrawer-paperAnchorLeft MuiDrawer-paperAnchorDockedLeft css-12i7wg6-MuiPaper-root-MuiDrawer-paper"
      >
        <div>
          <div
            class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
          >
            <div
              class="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap css-8u39c-MuiTypography-root"
            >
              Codeflix
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
          />
          <ul
            class="MuiList-root MuiList-padding css-h4y409-MuiList-root"
          >
            <a
              href="/"
              style="text-decoration: none;"
            >
              <li
                class="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
              >
                <div
                  class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
                  role="button"
                  tabindex="0"
                >
                  <div
                    class="MuiListItemText-root css-tlelie-MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                    >
                      Categories
                    </span>
                  </div>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </div>
              </li>
            </a>
            <a
              href="/cast-members"
              style="text-decoration: none;"
            >
              <li
                class="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
              >
                <div
                  class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
                  role="button"
                  tabindex="0"
                >
                  <div
                    class="MuiListItemText-root css-tlelie-MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                    >
                      Cast Members
                    </span>
                  </div>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </div>
              </li>
            </a>
            <a
              href="/genres"
              style="text-decoration: none;"
            >
              <li
                class="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
              >
                <div
                  class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
                  role="button"
                  tabindex="0"
                >
                  <div
                    class="MuiListItemText-root css-tlelie-MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                    >
                      Genres
                    </span>
                  </div>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </div>
              </li>
            </a>
            <a
              href="/videos"
              style="text-decoration: none;"
            >
              <li
                class="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
              >
                <div
                  class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
                  role="button"
                  tabindex="0"
                >
                  <div
                    class="MuiListItemText-root css-tlelie-MuiListItemText-root"
                  >
                    <span
                      class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                    >
                      Videos
                    </span>
                  </div>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </div>
              </li>
            </a>
          </ul>
        </div>
      </div>
    </div>
  </nav>
</DocumentFragment>
`;
